docs/rules/newline-per-chained-call.md

```diff
@@ -50,18 +50,20 @@ d3
 
 ## Rule Details
 
-This rule checks and reports the chained calls if there are no new lines after each call or deep member access. Computed property accesses such as `instance[something]` are excluded.
+This rule requires a newline after each call in a method chain or deep member access. Computed property accesses such as `instance[something]` are excluded.
 
 ## Options
 
-The rule takes a single option `ignoreChainWithDepth`. The level/depth to be allowed is configurable through `ignoreChainWithDepth` option. This rule, in its default state, allows 2 levels.
+This rule has an object option:
 
-* `ignoreChainWithDepth` Number of depths to be allowed (Default: `2`).
+* `"ignoreChainWithDepth"` (default: 2) allows chains up to a specified depth.
```

Comment:
What do you think about 2 versus `2` enclosed in backticks.
Not a big difference to me, but someone recently submitted a pull request to add backticks in similar situation.